kitchen/tests/kitchen/conftest.py

```diff
@@ -37,22 +40,29 @@ def __init__(self):
     def await_server(self, max_wait_seconds=60):
         @tenacity.retry(stop=tenacity.stop_after_delay(max_wait_seconds), wait=tenacity.wait_fixed(1))
         def await_helper():
-            assert requests.get(self.url())
+            assert requests.get(self.url(), verify='.')
         await_helper()
         logging.info(f'Kitchen server is running on {self.hostname}:{self.port}')
 
-    def url(self, path='/', scheme='http'):
+    def url(self, path='/'):
         assert path.startswith('/')
-        return f'{scheme}://{self.hostname}:{self.port}{path}'
+        return f'{self.scheme}://{self.hostname}:{self.port}{path}'
 
     def kill(self):
         if self.__server_process:
             self.__server_process.terminate()
             logging.info(f'Kitchen server has been killed')
 
-@pytest.fixture(scope="session", autouse=True)
+@pytest.fixture(scope="session")
 def kitchen_server(request):
     """Manages an instance of the Kitchen test app server."""
     server = KitchenServer()
```

Comment:
Explicitly test ssl=False (instead of the relying on the default False). Ensures that we continue to test HTTP even if we change the default. (Or, get rid of the default ssl=False and be explicit everywhere.)